test/style_trans_23_to_30.js

```diff
@@ -1153,24 +1153,45 @@ describe('cartocss transformation from 2.3.x to 3.0.x', function() {
                 '#points {',
                 '  marker-fill: #fee5d9;',
                 '  [ scalerank = 6 ] {',
-                '    marker-fill: #fcae91;',
-                '    marker-clip: true',
+                '    marker-fill: #fcae91',
                 '  }',
                 '  [ scalerank = 8 ] {',
-                '    marker-fill: #fb6a4a;',
-                '    marker-clip: true',
+                '    marker-fill: #fb6a4a',
                 '  }',
                 '  [ scalerank = 4 ] {',
-                '    marker-fill: #de2d26;',
-                '    marker-clip: true',
+                '    marker-fill: #de2d26',
                 '  }',
                 '  [ scalerank = 10 ] {',
-                '    marker-fill: #a50f15;',
-                '    marker-clip: true',
+                '    marker-fill: #a50f15',
                 '  }',
                 '  marker-clip: true',
                 '}',
             ].join('\n')
+        }, { // see: https://github.com/CartoDB/grainstore/issues/136
+            description: 'should not add defaults when parent has symbolizer already defined',
+            input: [
```

Comment:
I think it's gonna work, but I would add another test with two nested conditions.

```
#layer {
  marker-width: 4;
  marker-clip: false;
  [pop_max > 0] {
    marker-clip: true;
    marker-width: 8;
    [pop_max > 100] {
      marker-width: 16;
    }
  }
}
```